.github/CODEOWNERS

```diff
@@ -32,13 +32,15 @@ tests/internal/test_module.py       @DataDog/debugger-python @DataDog/apm-core-p
 
 # ASM
 ddtrace/appsec/                     @DataDog/asm-python
+ddtrace/contrib/subprocess/         @DataDog/asm-python
```

Comment:
I would only make the appsec specific code (which seems to be only the automatic enablement) as owned differently

```suggestion
ddtrace/contrib/subprocess/         @DataDog/asm-python
```

Reply:
Right now all the code on that directory is only for implementing an ASM RFC so I think it makes sense for it to have asm ownership, once APM or other team contributes I would make sense to make it multi owned or owned by APM to have a single entry point.